lib/eml2html/converter.rb

```diff
@@ -55,6 +56,33 @@ def save_zip(options = {})
       end
     end
 
+    def get_attachments(include_in_doc=false)
+      attachments = []
+      if include_in_doc
+        attachments = @attachments
+      else
+        @attachments.each do |a|
+          unless @cids_in_body.includes(a.cid)
+            attachments.push(a)
+          end
+        end
+      end
+      attachments
+    end
+
+    def get_html
```

Comment:
I don't see any point of having this.

Reply:
I'll just make html_body public, and then we wont need this.